### ISSUE TITLE
Support `picture` tag for markdown pages

### ISSUE THREAD

**Issue by ilfa** [dart-lang/pub-dev#5923]
Hi!

Can you keep `picture` tag for a markdown pages instead of omitting it?
 * Here is [the readme on pub.dev](https://pub.dev/packages/fpjs_pro_plugin/versions/1.0.4) without tag
 * Here is the same [readme on GitHub](https://github.com/fingerprintjs/fingerprintjs-pro-flutter/blob/1.0.4/README.md)
 * Here is the [GitHub documentation page](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#specifying-the-theme-an-image-is-shown-to) with the recommendations for using `picture` tag.

**Comment by isoos** [comment]
@jonasfj: should we add this to `package:sanitize_html`?

**Comment by ilfa** [comment]
I can do pr in `package:sanitize_html` if it will help.

**Comment by jonasfj** [comment]
Yes, I'm down with adding this to `package:sanitize_html`, feel free to make a PR.

-------

But I don't see this in:
https://github.github.com/gfm/
OR:
https://github.com/gjtorikian/html-pipeline/blob/main/lib/html/pipeline/sanitization_filter.rb

I'm not inclined to support tags not specified in GFM spec or `sanitization_filter.rb`, maybe github is now using a different sanitization library -- but then we should hopefully be able to find some canonical reference documentation or code for this behavior.

Linking to [github user documentation](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#specifying-the-theme-an-image-is-shown-to) does not seem like a good idea.

**Comment by chaudharydeepanshu** [comment]
Opened google/dart-neats#383 for the `sanitize_html` side. It mirrors `html-pipeline`'s current allowlist, which already lists `picture` / `source` / `source → srcset` (the referenceable source asked for above). cc @sigurdm. pub.dev's own `markdown.dart` would still need its rewriter + image proxy to handle `source[srcset]`; that's a separate follow-up.
